app/helpers/quadicon_helper.rb

```diff
@@ -155,7 +155,7 @@ def render_quadicon(item, options = {})
   def quadicon_builder_factory(item, options)
     case quadicon_builder_name_from(item)
     when 'service', 'service_template', 'service_ansible_tower', 'service_template_ansible_tower'
-      render_service_quadicon(item, options)
+      render_service_quadicon(item, options.merge!(:url => "/service/explorer/s-#{item.id}"))
```

Comment:
This mechanism is not pluggable :-(

I wonder how we can generalize this. Instead adding lines to this file, I'd rather have some decorator or a quadicon helper class for each entity that supports quadicons.

But that is a work for a different PR.

Reply:
I understand.
But `/service/explorer/s-#{item.id}` seems like the way to preselect a tree node in the service explorer.
So probably more generalization around that area on how to preselect nodes in an explorer view?

Reply:
> So probably more generalization around that area on how to preselect nodes in an explorer view?

Guess that is where @skateman 's recent work on the angularized tree component should bring us to.